modules/020-deckhouse/docs/README.md

```diff
@@ -50,3 +50,29 @@ Foe example: if you cluster have a lot of `NodeGroup` resources, it will take so
 #### Manual release deployment
 If you have a [manual update mode](usage.html#manual-update-confirmation) enabled and have a few Pending releases,
 you can approve them all at once. In that case Deckhouse will update in series keeping a release order and changing their status during the update.
+
+
+#### Pin a release
+Release pinning could be necessary if you want to hold a Deckhouse update for some reason.
+
+There are 3 options to pin a release:
+- Set a [manual update mode](usage.html#manual-update-confirmation).
```

Comment:
This link doesn't exist: https://github.com/deckhouse/deckhouse/blob/deckhouse-pin-image/modules/020-deckhouse/docs/usage.html#manual-update-confirmation
Is it ok?

Reply:
Yes, It will work only on the site. These anchors become actual only after web site building